### PR TITLE
.php_cs - add few rules

### PR DESCRIPTION
**Pull request by keradus**
Add few rules to PHP CS Fixer configuration that are already followed.

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -37,12 +38,14 @@ $config->fixers(
         'function_call_space',
         'function_declaration',
         'indentation',
+        'join_function',
```
Comment (Ocramius) [inline .php_cs]: Maybe we should bump the `composer.json` version of the fixer, to avoid regressions?

**Comment by gianarb** [comment]
@Ocramius with other CR team developers have you thinking a set of common roles for cs-fix? (thoughts after split :P )

File: .php_cs (hunk 1 above)
Comment (keradus) [inline .php_cs]: To be honest I don't get what's on your mind.
In composer there is latest stable version of PHP CS Fixer.
And what regression?

**Comment by keradus** [comment]
@gianarb that would be great. But currently we do not know which rules are followed in actual codebase - they are followed in most cases, but to turn then on they usually need little fixing to made Travis happy about them.

**Comment by gianarb** [comment]
Np for this PR.. It's a general question :+1: 
Sorry for this intrusion :smile:

**Comment by Ocramius** [comment]
@keradus thanks, merged after some further thoughts about it

**Pull request merged**